gerrymandering/gerrymandering.ipynb

```diff
@@ -47,20 +47,24 @@
    "source": [
     "import holoviews as hv\n",
     "from holoviews import opts\n",
+    "import hvplot.dask  # noqa\n",
+    "import hvplot.pandas  # noqa\n",
     "import geoviews as gv\n",
     "import datashader as ds\n",
     "import dask.dataframe as dd\n",
+    "import geopandas as gpd\n",
     "from cartopy import crs\n",
     "\n",
-    "from holoviews.operation.datashader import datashade\n",
+    "# from holoviews.operation.datashader import datashade\n",
     "\n",
     "hv.extension('bokeh')\n",
     "\n",
     "opts.defaults(\n",
-    "    opts.Points(apply_ranges=False, ),\n",
+    "    # opts.Points(apply_ranges=False, ),\n",
     "    opts.RGB(width=1200, height=682, xaxis=None, yaxis=None, show_grid=False),\n",
     "    opts.Shape(fill_alpha=0, line_width=1.5, apply_ranges=False, tools=['tap']),\n",
-    "    opts.WMTS(alpha=0.5)\n",
+    "    opts.WMTS(alpha=0.5),\n",
+    "    opts.Tiles(alpha=0.5),\n",
```

Comment:
Why set alpha on both WMTS (presumably a subclass of Tiles) and on Tiles?

Reply:
The alpha was initially (before my changes) on `opts.WMTS` because a `gv.WMTS` element was explicitly created. In this update, the tile layer is created from hvPlot with `tiles=True`, but without `geo=True`. The tiles in the overlay are a `hv.Tiles` element. `gv.WMTS` is **not** a subclass of `hv.Tiles`.
I should have commented out `opts.WMTS(alpha=0.5)`.